extension/test/unit/cart/deleteCart-test.js

```diff
@@ -0,0 +1,51 @@
+const assert = require('assert')
+const {PRODUCT, COUPON} = require('../../../common/consts')
+const executeStep = require('../../../cart/deleteCart')
+
+describe('deleteCart after checkout', () => {
+  const cartStorageName = 'user'
+
+  const cart = [
+    {
+      id: 'qwerty123',
+      productId: 'QWERTY123',
+      quantity: 1,
+      type: PRODUCT
+    },
+    {
+      id: '10off',
+      productId: '10off',
+      code: '10off',
+      type: COUPON,
+      quantity: 1
+    }
+  ]
+  const context = {
+    log: {
+      error: () => {}
+    },
+    storage: {
+      user: {}
+    }
+  }
+
+  it('Should backup cart and remove original cart', async () => {
```

Comment:
This is too much for a single test. If this fails, you won't be able to tell what is actually broken. I think it can be split into at least four tests (only one assert per test).